ui/app/helpers/supported-auth-backends.js

```diff
@@ -76,6 +76,13 @@ const SUPPORTED_AUTH_BACKENDS = [
     displayNamePath: ['metadata.org', 'metadata.username'],
     formAttributes: ['token'],
   },
+  {
+    type: 'certificate',
+    typeDisplay: 'Certificate',
```

Comment:
```suggestion
    typeDisplay: 'TLS certificate',
```

pour reprendre le terme officiel de la doc.